src/Molecules/Header/Header.stories.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react'
+import { MemoryRouter as Router, Route } from 'react-router'
+import { action } from '@storybook/addon-actions'
+import {  } from '@storybook/addon-knobs'
+import { Heart, Star, Home } from 'styled-icons/feather'
+
+import Header from './index'
+import ButtonGroup from '../../Atoms/ButtonGroup'
+import Icon from '../../Atoms/Icon'
+import StatusChip from '../../Atoms/StatusChip'
+import Tooltip from '../../Atoms/Tooltip'
+import TableList from '../TableList'
+
+import markdown from './README.md'
+
+export default {
+  title: 'Molecules/Header',
+}
+
+const SpacedTableList = () => <TableList style={{ marginTop: 10 }} />
+
+export const header = () => {
+  const [subFeature, setSubFeature] = useState('Header')
+  const handleSave = value => new Promise((resolve, reject) => setTimeout(() => {
+    const success = value && !/\W/.test(value.replace(/\s/g, ''))
+    action('saved')(success)
+
+    if (success) {
+      resolve()
+    } else {
+      window.alert('error')
+      reject(new Error('invalid'))
```

Comment:
Do you think you can force an error in the sb story?

Reply:
Yes it can be

https://github.com/rgsystemes/lib-ui/blob/4ddf236b4f982dd9583bc9fcab5b4ff08d3cf3df/src/Molecules/Header/Header.stories.jsx#L25